Mathlib/Data/ZMod/Basic.lean

```diff
@@ -1433,8 +1445,10 @@ end lift
 end ZMod
 
 section Module
-variable {S G : Type*} [AddCommGroup G] {n : ℕ} [Module (ZMod n) G] [SetLike S G]
-  [AddSubgroupClass S G] {K : S} {x : G}
+variable {n : ℕ} {S G : Type*} [AddCommGroup G] [SetLike S G] [AddSubgroupClass S G] {K : S} {x : G}
+
+section general
```

Comment:
Sorry for being unclear. I meant putting the module docstring in this section, so that it stays close to the `ZModModule` lemmas, and so that someone looking at those lemmas will easily find it.

The text looks good. Can you please move it here and merge afterwards?

bors d+